src/server/utils.rs

```diff
@@ -0,0 +1,34 @@
+/*
+ * server/utils.rs
+ *
+ * deepwell - Database management and migrations service
+ * Copyright (C) 2019 Ammon Smith
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// utilities used by other server methods
```

Comment:
The comment isn't specific enough to tell you anything that the name `utils.rs` doesn't imo